infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/query/parsing/impl/InfinispanRemoteQueryStringBuilderHelper.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.infinispanremote.query.parsing.impl;
+
+import java.util.List;
+
+/**
+ * Helper class to append single or multiple values to a {@link StringBuilder}
+ *
+ * @author Fabio Massimo Ercoli
+ */
+public class InfinispanRemoteQueryStringBuilderHelper {
```

Comment:
This class is not used anywhere

Reply:
I believe that it is used in the classes:
* InfinispanRemoteQueryRendererDelegate
* InfinispanRemoteComparisonPredicate
* InfinispanRemoteLikePredicate
* InfinispanRemoteRangePredicate